package.json

```diff
@@ -2,13 +2,18 @@
   "name": "focus-trap",
   "version": "2.4.3",
   "description": "Trap focus within a DOM node.",
-  "main": "index.js",
+  "main": "dist/focustrap.min.js",
```

Comment:
I'd rather leave the unminified version as the file that people get if they `require` the package. Do you think there's a reason to prefer the minified version?

Reply:
Actually you are right, I'll twist it.